.travis.yml

```
@@ -7,18 +7,15 @@ addons:
     sources:
       - ubuntu-toolchain-r-test
     packages:
-      - gcc-snapshot
+      - g++-9
       - doxygen
 
 install:
   - pip install --user conan cmake
 
 
 script:
-  - CXX=/usr/bin/gcc-10 CC=/usr/bin/g++-10 cmake -D ENABLE_COVERAGE:BOOL=TRUE . 
-  - cmake --build . -- -j2 
-  - ctest -j2
+  - CC=gcc CXX=g++ cmake -D ENABLE_COVERAGE:BOOL=TRUE .
+  - cmake --build . -- -j
+  - ctest -j
```

I would keep the `-j2`.
`-j` means infinite jobs.
```suggestion
  - cmake --build . -- -j2
  - ctest -j2
```

---

Okay, sure. In my experience make typically runs `nproc` jobs when specifying `-j` without any value.